### PR TITLE
jsclasses.org

### PR DESCRIPTION
**Pull request by galeksandrp**

### CODE REVIEW

File: src/chrome/content/rules/Jsclasses.org.xml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<!--files. refused-->
```
Comment (J0WI) [inline src/chrome/content/rules/Jsclasses.org.xml]: mismatch on safe.jsclasses.net

File: src/chrome/content/rules/Jsclasses.org.xml [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+<ruleset name="Jsclasses.org" platform="mixedcontent">
+	<target host="jsclasses.org" />
+	<target host="www.jsclasses.org" />
+	<target host="jsclasses.net" />
+	<target host="www.jsclasses.net" />
+	
+	<rule from="^http://(www\.)?jsclasses\.net/"
+		to="https://$1jsclasses.org/" />
```
Comment (J0WI) [inline src/chrome/content/rules/Jsclasses.org.xml]: please document why

**Comment by galeksandrp** [comment]
Done.

**Comment by J0WI** [comment]
What about https://github.com/EFForg/https-everywhere/pull/5833#discussion_r72720864?

**Comment by galeksandrp** [comment]
Added.

**Comment by J0WI** [comment]
Thanks!

**Pull request merged**